### PR TITLE
Bugfixes for extra team members

### PR DESCRIPTION
**Pull request by awisema**

### CODE REVIEW

File: app/controllers/health/patient_controller.rb [hunk 1]
```diff
@@ -22,8 +22,8 @@ def index
 
     def update
       @patient.update(patient_params)
-      @patient.build_team_member!(Health::Team::CareCoordinator, patient_params[:care_coordinator_id], current_user) if patient_params[:care_coordinator_id].present?
-      @patient.build_team_member!(Health::Team::Nurse, patient_params[:nurse_care_manager_id], current_user) if patient_params[:nurse_care_manager_id].present?
+      @patient.build_team_member!(Health::Team::CareCoordinator, patient_params[:care_coordinator_id].to_i, current_user) if patient_params[:care_coordinator_id].present?
```
Comment (eanders) [inline app/controllers/health/patient_controller.rb:25]: Throwing `to_i` on here will turn `nil` into `0` is that intentional?

File: app/models/health/patient.rb [hunk 2]
```diff
@@ -917,7 +917,9 @@ def name
 
     def build_team_member!(team_member_class, team_member_user_id, current_user)
       user = User.find(team_member_user_id)
-      team_member = team_member_class.where(patient_id: id, email: user.email).first_or_initialize
+      team_member = team_member_class.where(at[:email].lower.eq(user.email.downcase).to_sql).
```
Comment (eanders) [inline app/models/health/patient.rb]: where does `at` get defined?  maybe you mean `team_member_class.arel_table`?

File: app/controllers/health/patient_controller.rb (hunk 1 above)
Comment (awisema) [reply app/controllers/health/patient_controller.rb:25]: Team members should only be built if there is one present.

File: app/models/health/patient.rb (hunk 2 above)
Comment (awisema) [reply app/models/health/patient.rb]: Yes, thank you! Never throw in one last tweak after testing.

File: app/models/health/patient.rb [hunk 3]
```diff
@@ -917,7 +917,9 @@ def name
 
     def build_team_member!(team_member_class, team_member_user_id, current_user)
       user = User.find(team_member_user_id)
-      team_member = team_member_class.where(patient_id: id, email: user.email).first_or_initialize
+      team_member = team_member_class.where(team_member_class.arel_table[:email].lower.eq(user.email.downcase).to_sql).
```
Comment (eanders) [inline app/models/health/patient.rb]: I believe you'll want to wrap this in `where(Arel.sql(...to_sql))` if you are calling `to_sql` on it.

**Pull request merged**